realsense2_camera/CMakeLists.txt

```diff
@@ -118,6 +118,7 @@ find_package(tf2_ros REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(diagnostic_updater REQUIRED)
 find_package(OpenCV REQUIRED COMPONENTS core)
+find_package(fastrtps REQUIRED)  # a workaround, till librealsense2 cmake config will be fixed
```

Comment:
This can cause issues, we make ROS users be dependent on this package.
I am not sure which platforms support it and also which ros-distros.
We can add it for now but must find a solution for it.

Reply:
I will open a JIRA and add its number beside this ticket with a "TODO"

Reply:
I see you already opened LRS-1203